plaso/output/formatting_helper.py

```diff
@@ -332,6 +348,30 @@ def _FormatUsername(self, event, event_data, event_data_stream):
     """
     return self._output_mediator.GetUsername(event_data)
 
+  def _FormatWindowsEventLogMessage(self, event, event_data, event_data_stream):
+    """Formats a Windows Event Log message field.
+
+    Args:
+      event (EventObject): event.
+      event_data (EventData): event data.
+      event_data_stream (EventDataStream): event data stream.
+
+    Returns:
+      str: Windows Event Log message field or None if not available.
+    """
+    message_string = None
+    if event_data.source_name and event_data.message_identifier:
+      windows_event_message = self._output_mediator.GetWindowsEventMessage(
+          event_data.source_name, event_data.message_identifier)
+      if windows_event_message:
+        try:
+          message_string = windows_event_message.format(*event_data.strings)
+        except IndexError:
+          # Unable to create the message string.
+          pass
```

Comment:
Should we still try to return something?
IIUC, here we do have a windows_event_message but just not found a way to format it

Reply:
ack, I'll add a todo to address this in a later change